features/cancel_buttons.feature

```diff
@@ -0,0 +1,119 @@
+Feature: Cancel buttons
+  As an author
+  I want to be able to cancel out of what im doing and be warned appropriately
+  Scenario: Cancel out of editing a draft question
+    Given I am logged in as test_author@gmail.com
+    And I have a Question with the content "What is your gender?" and the description "This is a question" and the type "MC"
+    When I go to the list of Questions
+    And I click on the menu link for the Question with the content "What is your gender?"
+    And I click on the option to Details the Question with the content "What is your gender?"
+    And I click on the "Edit" button
+    And I click on the "Cancel" button
+    Then I should see "Question Details"
+    And I should see "Name: What is your gender?"
+    And I should see "Version: 1"
+    And I should see "History"
+    And I should see "Comments:"
+    And I should not see "Revise Question"
+
+  # Scenario: Cancel out of editing a draft form
+  # Given I am logged in as test_author@gmail.com
+  # Scenario: Cancel out of editing a draft response set
+  # Given I am logged in as test_author@gmail.com
```

Comment:
Talked to kevin - these are tests that need to be implemented in the future by stories that have not been implemented yet; not going to have this be a blocker of this PR